zivid_samples/src/sample_capture_and_save_frame.cpp

```diff
@@ -0,0 +1,65 @@
+#include <zivid_camera/SettingsAcquisitionConfig.h>
+#include <zivid_camera/CaptureAndSaveFrame.h>
+#include <dynamic_reconfigure/Reconfigure.h>
+#include <dynamic_reconfigure/client.h>
+#include <ros/ros.h>
+#include <ros/package.h>
+
+#define CHECK(cmd)                                                                                                     \
+  do                                                                                                                   \
+  {                                                                                                                    \
+    if (!cmd)                                                                                                          \
+    {                                                                                                                  \
+      throw std::runtime_error{ "\"" #cmd "\" failed!" };                                                              \
+    }                                                                                                                  \
+  } while (false)
+
+namespace
+{
+const ros::Duration default_wait_duration{ 30 };
+
+void capture_and_save_frame()
+{
+  ROS_INFO("Calling capture_and_save_frame service");
+  zivid_camera::CaptureAndSaveFrame capture_and_save_frame;
+  std::string samples_path = ros::package::getPath("zivid_samples");
+  std::string file_path = samples_path + "/capture_cpp.zdf";
+  ROS_INFO("and save frame to: %s", file_path.c_str());
```

Comment:
This will be a separate log line, so it looks better with just "Saving frame to %s" or something

Reply:
I will remove it. Didn't add it in the Python code, and the service will log it.